plugins/modules/jenkins_plugin.py

```diff
@@ -195,6 +195,29 @@
     url_password: p4ssw0rd
     url: http://localhost:8888
 
+#
+# Example of how to authenticate with serverless deployment
+#
+- name: Update plugins on ECS Fargate Jenkins instance
+  community.general.jenkins_plugin:
+    # plugin name and version
+    name: ws-cleanup
+    version: '0.45'
+    # Jenkins home path mounted on ec2-helper VM (example)
+    jenkins_home: "/mnt/{{ jenkins_instance }}"
+    # default local EC2 user that owns the mount
+    owner: ubuntu
+    group: ubuntu
```

Comment:
I just noticed one more thing:

> That way we can provide user/group of a default EC2 local user (which has the same uid/gid 1000 as the default jenkins user inside the container)

Why not simply providing the UID/GID directly?
```suggestion
    owner: 1000
    group: 1000
```
That way you don't have to first figure out which local user has the same UID/GID as the user in the container.

Reply:
That's a good idea and I've tried it - it works.
Actually, I've used names cause the docs said so :)
Now I've updated that too (see above)

Reply:
> I just noticed one more thing:
> 
> > That way we can provide user/group of a default EC2 local user (which has the same uid/gid 1000 as the default jenkins user inside the container)
> 
> Why not simply providing the UID/GID directly?
> ```suggestion
>     owner: 1000
>     group: 1000
> ```
> That way you don't have to first figure out which local user has the same UID/GID as the user in the container.

Also, since this works woth 1000/1000 combo, do you agree defaults should be changed to 1000 instead of 'jenkins'?

Reply:
Changing the default will break a lot of existing setups, where the user `jenkins` has another UID/GID than 1000. (Jenkins doesn't always run in the default container.) This would break backwards compatibility and is only allowed with a long deprecation period.

So let's better keep the default. Updating the documentation is definitely a good idea, though!

Reply:
> Changing the default will break a lot of existing setups, where the user `jenkins` has another UID/GID than 1000. (Jenkins doesn't always run in the default container.) This would break backwards compatibility and is only allowed with a long deprecation period.
> 
> So let's better keep the default. Updating the documentation is definitely a good idea, though!

Agreed, thought about that too as soon as I clicked "send". Ok so we stick to my doc changes only